CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Enhancements
```

Comment:
Probably a `Feature` and I'd add a note about a breaking change that can affect the release health data.
I'm wondering if this requires a major bump even.